Gemfile.lock

```diff
@@ -48,32 +47,33 @@ GEM
     method_source (0.9.2)
     mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.2)
+    money (6.13.4)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.1)
+    nokogiri (1.10.4)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.6)
+    rack (2.0.7)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.4)
+    rails-html-sanitizer (1.2.0)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.2.1)
-      actionpack (= 5.2.2.1)
-      activesupport (= 5.2.2.1)
+    railties (6.0.0)
+      actionpack (= 6.0.0)
+      activesupport (= 6.0.0)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.19.0, < 2.0)
-    rake (12.3.2)
+      thor (>= 0.20.3, < 2.0)
+    rake (12.3.3)
     thor (0.20.3)
     thread_safe (0.3.6)
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    zxcvbn-ruby (0.1.2)
+    zeitwerk (2.1.9)
+    zxcvbn-ruby (1.0.0)
```

Comment:
### [zxcvbn-ruby](http://github.com/envato/zxcvbn-ruby)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.1.2 → 1.0.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **16** commits. See the full changes on [the compare page](https://github.com/envato/zxcvbn-ruby/compare/v0.1.2...v1.0.0).

These are the first 10 commits:
- (df2ae63) [Adding license info to the gemspec.](https://github.com/envato/zxcvbn-ruby/commit/df2ae634750f360f12d517e9b0aeb3f6cb0e3218)
- (431f514) [Add a "FeedbackGiver" and "Feedback" class for feedback](https://github.com/envato/zxcvbn-ruby/commit/431f514079a2770009cf6bf5d8b239cb7ce06b40)
- (1a4fb8a) [Improve descriptiveness of some comments](https://github.com/envato/zxcvbn-ruby/commit/1a4fb8aa2924963636f5670bf5b48833effb8d8b)
- (1e23e5f) [Remove branches for unsupported feedback indicators](https://github.com/envato/zxcvbn-ruby/commit/1e23e5f79a02a564b23a6bc0c27bcc2e6930f31e)
- (25a4415) [Remove handler for the unsupported multi-character repeat ma…](https://github.com/envato/zxcvbn-ruby/commit/25a4415a8dec54517dfc9a47de458529d8661bef)
- (9f46af7) [Cover the rest of the FeedbackGiver in specs](https://github.com/envato/zxcvbn-ruby/commit/9f46af79ca38d56a35aa2bf52bd46c7872ccad70)
- (e47b763) [Remove Ruby 2.4isms because 1.9 is still a target here](https://github.com/envato/zxcvbn-ruby/commit/e47b763fec59ef17da1b5fe7b6497cda59f95478)
- (134c4a3) [Update Readme and pretty-print examples](https://github.com/envato/zxcvbn-ruby/commit/134c4a3f0b1f16e02f0ea8cc9a0e0d7d5192b81c)
- (fc69afd) [Merge pull request #21 from reiz/patch-1](https://github.com/envato/zxcvbn-ruby/commit/fc69afd08f5fb0c8361d52461750b716c925cb11)
- (e156c7a) [adding in supported ruby versions (2.3 +)](https://github.com/envato/zxcvbn-ruby/commit/e156c7add94b5cdeb2f07531cb22168b995be365)


</details>